src/App.js

```diff
@@ -62,7 +62,7 @@ const App = () => {
         <ion-icon name="search-sharp"></ion-icon>
       </div>
       <Modal
-        className="bg-primary border-2 color-secondary max-w-xl mx-auto my-24 px-2 py-1"
+        className="bg-primary border-2 border-gold color-secondary max-w-xl mx-auto my-24 px-2 py-1"
```

Comment:
This takes advantage of the change made in [this commit](6d9f5eff358a9a3bf1471e3a1f7f863d19f28892) in `tailwind.config.js`. The border of the modal window will always be gold regardless of whether users have a day or night theme active, matching the [Figma](https://www.figma.com/file/jeyLWmyn78DjqSmbWcJOEd/Tide?node-id=0%3A1).

This also saves us from having to create a new CSS class in `styles.css` for the sole purpose of adding a border to the redirect notice modal window that is always yellow, thereby reducing clutter.

Reply:
we should rename border-gold to be consistent with `bg-primary` class. Maybe `border-primary`?

Reply:
We want the border to always be gold, right?

In the the [Figma](https://www.figma.com/file/jeyLWmyn78DjqSmbWcJOEd/Tide?node-id=0%3A1), the modal-window doesn't have a visible border during the day, so my solution was to always have a gold border.

This results in the gold modal-window having a gold border if the day theme is active, appearing borderless, while at night, the gold border appears, matching what's in the Figma.

Reply:
If we do `border-primary` and the user has a night theme active, I believe that causes the modal-window to appear borderless, whereas it should have a yellow border when the night theme is active.

This is where I was trying to change the configuration files so we could have an "always-gold" color for things that we don't want to change colors.

Reply:
Tying class names to color is an antipattern in this case. We should be able to change the primary color to blue just by changing the css variable. Also to be consistent, I've updated the figma design to now have a dark border in light mode. This should simplify the implementation.

Reply:
> Tying class names to color is an antipattern in this case. We should be able to change the primary color to blue just by changing the css variable. Also to be consistent, I've updated the figma design to now have a dark border in light mode. This should simplify the implementation.

It definitely did simplify things! That rule is no longer needed in `tailwind.config.js` as a result of the update you made, so [I removed it with the latest commit at the time of this writing](https://github.com/amilajack/freesound-player/pull/99/commits/4a47f5fec9bc5527ce5d987f2bcaec0a1248049e#diff-ddc1067f080930bc48865f4660316a14b21bbc9f9ba85975ce8479342b983d2e).